### ISSUE TITLE
ATACFlow

### ISSUE THREAD

**Issue by wzthu** [Bioconductor/Contributions#505]
Update the following URL to point to the GitHub repository of
the package you wish to submit to _Bioconductor_

- Repository: https://github.com/wzthu/ATACFlow

Confirm the following by editing each check box to '[x]'

- [x] I understand that by submitting my package to _Bioconductor_,
  the package source and all review commentary are visible to the
  general public.

- [x] I have read the _Bioconductor_ [Package Submission][2]
  instructions. My package is consistent with the _Bioconductor_
  [Package Guidelines][1].

- [x] I understand that a minimum requirement for package acceptance 
  is to pass R CMD check and R CMD BiocCheck with no ERROR or WARNINGS. 
  Passing these checks does not result in automatic acceptance. The 
  package will then undergo a formal review and recommendations for 
  acceptance regarding other Bioconductor standards will be addressed.

- [x] My package addresses statistical or bioinformatic issues related
  to the analysis and comprehension of high throughput genomic data.

- [x] I am committed to the long-term maintenance of my package. This
  includes monitoring the [support site][3] for issues that users may
  have, subscribing to the [bioc-devel][4] mailing list to stay aware
  of developments in the _Bioconductor_ community, responding promptly
  to requests for updates from the Core team in response to changes in
  _R_ or underlying software.

I am familiar with the essential aspects of _Bioconductor_ software
management, including:

- [x] The 'devel' branch for new packages and features.
- [x] The stable 'release' branch, made available every six
      months, for bug fixes.
- [x] _Bioconductor_ version control using [Git][5]
  (optionally [via GitHub][6]).

For help with submitting your package, please subscribe and post questions
to the [bioc-devel][4] mailing list.

[1]: https://bioconductor.org/developers/package-guidelines/
[2]: https://bioconductor.org/developers/package-submission/
[3]: https://support.bioconductor.org
[4]: https://stat.ethz.ch/mailman/listinfo/bioc-devel
[5]: http://bioconductor.org/developers/how-to/git/
[6]: http://bioconductor.org/developers/how-to/git/sync-existing-repositories/

**Comment by wzthu** [comment]
Hi @hpages ,
Our package depends on QuasR package.
The QuasR does not support win32 platform.

Thus, we follow QuasR package and add .BBSoptions file to set "UnsupportedPlatforms" like this:

UnsupportedPlatforms: win32

However, the windows sever still builds and checks our package on win32 platform. 
The ERROR comes from this problem. 

Should we do something else to fix this ERROR?

Thank you very much!

**Comment by hpages** [comment]
Hi Zheng Wei,
Thanks for your submission. The .BBSoptions file is honored by our main build system (a.k.a. BBS, in charge of the daily builds here https://bioconductor.org/checkResults/devel/bioc-LATEST/), but I don't know if that's the case for the Single Package Builder. Maybe Lori (@lshep) could clarify. Let's ignore this error. I'm going to take a first look at ATACFlow today and will get back to you.
Cheers,
H.

**Comment by wzthu** [comment]
Thank you, @hpages . I am looking forward to your suggestion.

**Comment by hpages** [comment]
Hi Zheng Wei,

I just realize now that the package is not about flow cytometry (as the "flow" prefix in the name kind of suggests). What about naming the package ATACpipe or ATACPipe? That would be less confusing.

Also I just added the ATACSeq and DNaseSeq terms to the official list of supported terms. I would suggest that you add these terms to your biocViews field.

Thanks!
H.

**Comment by wzthu** [comment]
Hi @hpages ,

Thank you for your suggestions.

You mentioned that "ATACpipe" may be a better package name. We quite agree with you. And, it is also the first name of this package. We have changed it back. 

Besides, I have added the ATACSeq and DNaseSeq to biocViews field following your suggestion.

It seems that the sever can not recognize the package name and repository name changing .
So I have resubmited it as new issue [ATACpipe #529](https://github.com/Bioconductor/Contributions/issues/529)
Will this affect our package to be included in bioc 3.6 release?

Thank you very much!

Zheng Wei
